API-8669 Document reports-api protocol improvements (#891)
livechat/livechat-public-docs: content/data-reporting/reports-api/v3.4/index.mdx

| Header         | Value              | Required |
| -------------- | ------------------ | -------- |
| `Content-Type` | `application/json` | Yes      |

<Section>
<Text>

## Available parameters and filters

| Parameter                                                             | Required | Data type | Notes                                                                                                                                                                                                                                |
| --------------------------------------------------------------------- | -------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| `distribution`                                                        | No       | `string`  | Allowed values: `hour`, `day`, `day-hours`, `month` or `year`. Defaults to `day`.                                                                                                                                                    |

> | `timezone`                                                            | No       | `string`  | IANA [Time Zone](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix). Defaults to the requester's timezone. When requester's timezone is not present then `filters.from` is parsed to get timezone.  |

Reviewer: ```suggestion
| `timezone`                                                            | No       | `string`  | IANA [Time Zone](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix). Defaults to the requester's timezone. When the requester's timezone isn't present, then `filters.from` is parsed to get the timezone.  |
```

> | `filters`                                                             | No       | `object`  | If not provided your report will be spanned last seven days.                                                                                                                                                                         |

Reviewer: ```suggestion
| `filters`                                                             | No       | `object`  | If none provided, your report will span the last seven days.                                                                                                                                                                         |
```

|                     |                                                                           |
| ------------------- | ------------------------------------------------------------------------- |
| **Method URL**      | `https://api.livechatinc.com/v3.4/reports/chats/agents_chatting_duration` |

> | **HTTP method**     | GET, POST                                                                 |

Reviewer 2: Since `POST` is recommended, it should go first

<Sample>

```shell
curl "https://api.livechatinc.com/v3.4/reports/chats/agents_chatting_duration" \
  -H 'Content-Type: application/json' \
  -H 'Authorization: Bearer <your_access_token>' \

>   -d '{}'

Reviewer 2: Don't we want to put any filters in the example?

Author: As Ola said filters example is presented below params table and i do not see huge value in duplicating this in each report

Reviewer 2: It just seems odd to me to describe the filters and then show the one use case which doesn't use them at all

Author: Added sample request payloads
